examples/how-tos/react-human-in-the-loop.ipynb

```diff
@@ -0,0 +1,342 @@
+    "\n",
+    "We can try resuming and we will see an error arise:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 73,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Error: Unknown Location\n",
+      " Please fix your mistakes.\n",
+      "-----\n",
+      "\n",
+      "[\n",
+      "  {\n",
+      "    name: 'get_weather',\n",
+      "    args: { location: 'San Francisco, California' },\n",
+      "    type: 'tool_call',\n",
+      "    id: 'call_MfIPKpRDXRL4LcHm1BxwcSTk'\n",
+      "  }\n",
+      "]\n",
+      "-----\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "for await (\n",
+    "    const { messages } of await agent.stream(null, {\n",
+    "      ...config,\n",
+    "      streamMode: \"values\",\n",
+    "    })\n",
+    "  ) {\n",
+    "    let msg = messages[messages?.length - 1];\n",
+    "    if (msg?.content) {\n",
+    "      console.log(msg.content);\n",
+    "    }\n",
+    "    if (msg?.tool_calls?.length > 0) {\n",
+    "      console.log(msg.tool_calls);\n",
+    "    }\n",
+    "    console.log(\"-----\\n\");\n",
+    "  }"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This error arose because our tool argument of \"SF, California\" is not a location our tool recognizes.\n",
+    "\n",
+    "Let's show how we would edit the tool call to search for \"San Francisco\" instead of \"SF, California\" - since our tool as written treats \"San Francisco, CA\" as an unknown location. We will update the state and then resume streaming the graph and should see no errors arise:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 74,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\n",
+      "  configurable: {\n",
+      "    thread_id: '1',\n",
+      "    checkpoint_ns: '',\n",
+      "    checkpoint_id: '1ef6638d-bfbd-61d0-8004-2751c8c3f226'\n",
+      "  }\n",
+      "}\n"
+     ]
+    }
+   ],
+   "source": [
+    "// First, lets get the current state\n",
+    "const currentState = await agent.getState(config);\n",
+    "\n",
+    "// Let's now get the last message in the state\n",
+    "// This is the one with the tool calls that we want to update\n",
+    "let lastMessage = currentState.values.messages[currentState.values.messages.length - 1]\n",
+    "\n",
+    "// Let's now update the args for that tool call\n",
+    "lastMessage.tool_calls[0].args = { location: \"San Francisco\" }\n",
+    "\n",
+    "// Let's now call `updateState` to pass in this message in the `messages` key\n",
+    "// This will get treated as any other update to the state\n",
+    "// It will get passed to the reducer function for the `messages` key\n",
+    "// That reducer function will use the ID of the message to update it\n",
```

Comment:
I would highlight the significance of the message id above outside of the code block too